.circleci/config.yml

```diff
@@ -122,15 +99,14 @@ workflows:
           name: "Build Docker Images"
           requires:
             - "Run Tests"
-
   Release:
     jobs:
-      - release:
+      - rok8s-scripts/github_release:
           filters:
             branches:
               ignore: /.*/
             tags:
-              only: /^v.*/
+              only: /.*/
```

Comment:
Just checking to make sure removing the "v" was intended. Not a blocker, I don't have an opinion either way.

Reply:
It was not intentional. Thanks!